### DISCUSSION TITLE
Support shorthand `<div {property} />` syntax

### DISCUSSION THREAD

**Discussion by farism** [mint-lang/mint#699] [Language Development]
In Mint, to pass a bunch of property variables to a component, you have to list out the key/value for each one.

```
component Main {
  fun render : Html {
    let disabled =
      false

    let onClick =
      (e : Html.Event) { "" }

    <button
      disabled={disabled}
      onClick={onClick}/>
  }
}
```

This can get cumbersome and noisy when you have many props.

It would be nice if Mint had the ability to do shorthand properties (like in Svelte)

```
component Main {
  fun render : Html {
    let disabled =
      false

    let onClick =
      (e : Html.Event) { "" }

    <button {disabled} {onClick} />
  }
}
```

**Comment by gdotdesign** [comment]
I would like to have a discussion about this before we implement anything like that, so converting this into a discussion.
